### ISSUE TITLE
Tableau web connector hard coded "hive" catalog in it

### ISSUE THREAD

**Issue by niyue** [prestodb/presto#5514]
I found Presto Tableau web connector hard coded "hive" catalog in it (tableau/presto-connector.html):
```
             var catalogClient = new StatementClient({
                    'catalog': 'hive',
                    'schema': 'default',
                     'query': 'show catalogs'
              },
        ......
        populateSchemas('hive');
````
This makes the Tableau web connector failed to work when the Presto is deployed without 'hive' connector configured, and Tableau web connector will always report error "Catalog hive does not exist" while users doesn't have hive configured at all. Probably using the 'system' catalog as default is better.

**Comment by martint** [comment]
I think @chrisunder is working on this.

**Comment by chrisunder** [comment]
Please see https://github.com/prestodb/presto/pull/5538 for my work on this issue.

**Comment by niyue** [comment]
@electrum I did fix this locally but don't send a PR because at that time I found the latest revision's web UI is changed heavily from 0.148 to 0.149  and I am not sure if someone is planning to revamp this completely.

@chrisunder Thanks.

**Comment by chrisunder** [comment]
I also have https://github.com/prestodb/presto/pull/5537 which is related to the UI changes

**Comment by nezihyigitbasi** [comment]
This can be closed as both #5538 and #5537 are merged.
